### PR TITLE
Double-width space should be used as search delimiter.

### PR DESCRIPTION
**Pull request by miya0001**

### CODE REVIEW

**Comment by miya0001** [comment]
Add tests to check `wp_mail` should send email as expected encoding.

File: tests/phpunit/test-wp-multibyte-patch.php [hunk 1]
```diff
@@ -42,12 +42,64 @@ function test_length_of_excerpt_rss_should_be_110()
...
+			's' => $_GET['s']
+		) );
+
+		$this->assertTrue( !! count( $q->posts ) );
+	}
+
+	/**
+	 * `wp_mail` should encode as expected
+	 */
+	public function test_wp_mail_should_be_send_as_iso_2022_jp()
```
Comment (ryotsun) [inline tests/phpunit/test-wp-multibyte-patch.php:67]: I guess `test_wp_mail_should_be_sent_as_iso_2022_jp` is better. 😅

File: tests/phpunit/test-wp-multibyte-patch.php [hunk 2]
```diff
@@ -42,12 +42,64 @@ function test_length_of_excerpt_rss_should_be_110()
...
+	{
+		$to = 'hello@example.com';
+		$subject = "こんにちは";
+		$message = "日本語のメール";
+		$result = wp_mail( $to, $subject, $message );
+
+		$mailer = tests_retrieve_phpmailer_instance();
+
+		$this->assertSame( 'hello@example.com', $mailer->get_recipient( 'to' )->address );
```
Comment (ryotsun) [inline tests/phpunit/test-wp-multibyte-patch.php:76]: Could you tell me the reason why you added this assertion?
To just verify if the email address is not affected?

File: tests/phpunit/test-wp-multibyte-patch.php [hunk 3]
```diff
@@ -42,12 +42,64 @@ function test_length_of_excerpt_rss_should_be_110()
...
+			$mailer->get_sent()->subject
+		);
+		$this->assertSame(
+			"こんにちは",
+			iconv_mime_decode( $mailer->get_sent()->subject, 0, 'UTF-8' )
+		);
+
+		// Body should be encoded as ISO-2022-JP
+		$this->assertSame(
+			"日本語のメール\n",
```
Comment (ryotsun) [inline tests/phpunit/test-wp-multibyte-patch.php:90]: [note] `\n` is inserted after sent a mail.

File: tests/phpunit/test-wp-multibyte-patch.php [hunk 4]
```diff
@@ -42,12 +42,64 @@ function test_length_of_excerpt_rss_should_be_110()
...
+
+	/**
+	 * `wp_mail` should encode as expected
+	 */
+	public function test_wp_mail_should_be_send_as_iso_2022_jp()
+	{
+		$to = 'hello@example.com';
+		$subject = "こんにちは";
+		$message = "日本語のメール";
+		$result = wp_mail( $to, $subject, $message );
```
Comment (ryotsun) [inline tests/phpunit/test-wp-multibyte-patch.php:72]: It could be 
```php
wp_mail( $to, $subject, $message );
```
because `$result` is not used.

**Review by ryotsun — COMMENTED** [review]
Reviewed.

File: tests/phpunit/test-wp-multibyte-patch.php (hunk 2 above)
Comment (miya0001) [reply tests/phpunit/test-wp-multibyte-patch.php:76]: Yes, it doesn't have many means... 😊

**Pull request merged**